client.py

```diff
@@ -0,0 +1,20 @@
+import requests
+
+URL = "vcm-3577.vm.duke.edu"
+
+
+name = requests.get(URL + "/name")
+print(name.json())
+
+
+hello = requests.get(URL + "/hello/Suyash")
+print(hello.json())
+
+
+data = {
+    "a": [2, 4],
+    "b": [5, 6]
+}
+p = requests.post(URL + "/distance", json=data)
+print(p.json())
+
```

Comment:
for this `client` program, it would've been better to structure it how we discussed early on in this class -- to have a `main` function, and follow the `if __name__ == "__main__"` convention. You could have also broken out the request code into helper functions for future reusability.